sklearn/preprocessing/_encoders.py

```diff
@@ -699,6 +699,26 @@ class OrdinalEncoder(_BaseEncoder):
     dtype : number type, default np.float64
         Desired dtype of output.
 
+    missing_value : object type or a list of object types, default "nan"
+        Which value(s) passed to ``transform()`` to be considered missing
+
+    handle_missing : string, default 'ignore'
+        How to handle missing values passed to ``transform()``
+
+        - 'ignore' : Allow missing values to pass through ``transform()`` and
+           and be returned as np.nan
+        - 'largest' : Encode missing values as largest ordinal category
+        - 'smallest' : Encode missing values as 0
+        - 'separate' : Encode missing values as -1
+
+    handle_unknown : string, default 'error'
+        Handle values not passed to ``fit()`` and not specified as missing
+
+        - 'error' : Unknown values raise error
+        - 'ignore' : Unknown values encoded as 0
```

Comment:
The encoded categories already start at 0, so I don't see how this is possible?

Reply:
BTW, the reason this keyword was not added to `OrdinalEncoder` initially (but it is added to `OneHotEncoder`) is exactly because there is no clear meaning of 'ignore' I think (at least when categories are encoded as 0...n). 

I also think this is quite independent from the actual missing value handling, so maybe leave this for separate issue/PR to discuss?

Reply:
Okay, you're right. I'll leave this one alone.